lib/cisco_node_utils/bgp_af.rb

```diff
@@ -160,6 +188,32 @@ def default_next_hop_route_map
       config_get_default('bgp_af', 'next_hop_route_map')
     end
 
+    def next_hop_route_policy
+      return nil if vrf_unsupported_xr?
+      config_get('bgp_af', 'next_hop_route_policy', @get_args)
+    end
+
+    def next_hop_route_policy=(route_policy)
+      fail_if_vrf_unsupported_xr('nexthop route-policy')
+      #      route_policy.strip!
+      if route_policy.empty?
+        state = 'no'
+        # Dummy route policy required if not configured.
+        if next_hop_route_policy.empty?
+          route_policy = 'dummy_routepolicy'
```

Comment:
I'd like to know if the team thinks that adding a dummy route_policy is ok, or is it better to fail (from the manifest) and add support for route-policy asap?

Good example is rd auto - we just failed until someone added it.